### DISCUSSION TITLE
Welcome to OCI-OcC-Fix Discussions!

### DISCUSSION THREAD

**Discussion by mosesman831** [mosesman831/OCI-OcC-Fix#1] [Announcements]
## You can share anything about OCI-OcC-Fix or generally about Oracle Cloud!

**Comment by m3n0** [comment]
i am trying to use your method to fix the current problem of full capacity of arm VM's
i am facing this errors 
please find the attached image 
![github_issue_arm](https://github.com/user-attachments/assets/34566d29-4af6-46a1-aef8-ce38e7c078b6)

**Reply by mosesman831** [reply]
Could you start an issue and also paste your python file, but do remove all the personal information from it.
